tests/foreman/api/test_contentmanagement.py

```diff
@@ -272,6 +273,7 @@ def test_positive_uploaded_content_library_sync(self):
         self.assertEqual(len(capsule_rpms), 1)
         self.assertEqual(capsule_rpms[0], RPM_TO_UPLOAD)
 
+    @upgrade
```

Comment:
This has already covered in upgrade workflow scenarios that's why we are able to install capsule and move forward.

Reply:
ok, we might need to come up with another decorator to tag the tests we want to run in z-stream.